src/SprykerEco/Zed/Heidelpay/Business/Adapter/Mapper/RequestToHeidelpay.php

```diff
@@ -47,9 +47,13 @@ public function map(HeidelpayRequestTransfer $requestTransfer, Request $heidelpa
 
         $heidelpayRequest->basketData(
             $requestTransfer->getCustomerPurchase()->getIdOrder(),
-            $requestTransfer->getCustomerPurchase()->getAmount(),
+            (string)$requestTransfer->getCustomerPurchase()->getAmount(),
```

Comment:
This is dangerous conversion, since it's not defined what kind of decimal separator will be generated

Reply:
Fixed